plugins/autosuspend.koplugin/main.lua

```diff
@@ -137,6 +137,12 @@ if Device:isKindle() then
             return
         end
 
+        -- Also causes problems when charging.
+        if self.is_charging then
```

Comment:
What's the precise way that's set? It kind of reads like you should just do the `PowerD:isCharging() and not PowerD:isCharged()` check right here.